packages/core/src/services/commits/merge.ts

```diff
@@ -35,6 +36,16 @@ export async function mergeCommit(
       )
     }
 
+    const recomputedResults = await recomputeChanges({ commitId }, tx)
+    if (recomputedResults.error) return recomputedResults
+    if (Object.keys(recomputedResults.value.errors).length > 0) {
+      return Result.error(
+        new LatitudeError(
+          'There are errors in the updated documents in this commit',
+        ),
+      )
+    }
```

Comment:
Mergeing will always recompute changes for two reasons:
- To always have ALL changes.
- Validate that none of the changes contain errors.